libnavigation-core/src/main/java/com/mapbox/navigation/core/MapboxNavigation.kt

```diff
@@ -655,18 +759,70 @@ class MapboxNavigation @VisibleForTesting internal constructor(
      *
      * @return a list of [DirectionsRoute]s
      */
-    fun getRoutes(): List<DirectionsRoute> = directionsSession.routes
+    @Deprecated(
+        "use #getNavigationRoutes() instead",
+        ReplaceWith(
+            "getNavigationRoutes().toDirectionsRoutes()",
+            "com.mapbox.navigation.base.route.toDirectionsRoutes"
+        )
+    )
+    fun getRoutes(): List<DirectionsRoute> = directionsSession.routes.toDirectionsRoutes()
+
+    /**
+     * Get a list of routes.
+     *
+     * If the list is not empty, the route at index 0 is the one treated as the primary route
+     * and used for route progress, off route events and map-matching calculations.
+     *
+     * @return a list of [NavigationRoute]s
+     */
+    fun getNavigationRoutes(): List<NavigationRoute> = directionsSession.routes
 
     /**
      * Requests an alternative route using the original [RouteOptions] associated with
      * [MapboxNavigation.setRoutes()] call and [Router] implementation.
      * @see [registerRouteAlternativesObserver]
      */
-    @JvmOverloads
-    fun requestAlternativeRoutes(callback: RouteAlternativesRequestCallback? = null) {
+    fun requestAlternativeRoutes() {
+        routeAlternativesController.triggerAlternativeRequest(null)
+    }
+
+    /**
+     * Requests an alternative route using the original [RouteOptions] associated with
+     * [MapboxNavigation.setRoutes()] call and [Router] implementation.
+     * @see [registerRouteAlternativesObserver]
+     */
+    fun requestAlternativeRoutes(callback: NavigationRouteAlternativesRequestCallback? = null) {
```

Comment:
we already have empty version of `requestAlternativeRoutes`. Should it be not optional then?
```suggestion
    fun requestAlternativeRoutes(callback: NavigationRouteAlternativesRequestCallback = null) {
```